WheelOfFortune/Prompt.cs

```diff
@@ -101,18 +102,19 @@ public static dynamic PromptAction(Action action)
             dynamic userInput=null;
             if (action.ActionTypeProperty == Action.ActionType.GuessLetterAction)
             {
-                Console.WriteLine("Start Guessing: Enter a letter:");
+                Console.Write("Start Guessing:");
+                Console.Write("Enter a letter:");
                 userInput = Console.ReadLine();
                 userInput = validateUserGuess(userInput);
                 userInput = userInput[0];
             }
             else if(action.ActionTypeProperty == Action.ActionType.SolvePuzzleAction)
             {
-                Console.WriteLine("What is your solution to the puzzle?");
+                Console.Write("What is your solution to the puzzle?");
```

Comment:
To stay consistent with the other prompts, could you add a \n to the beginning of this prompt so that there will be space between this prompt and what ever is printed before it?